### ISSUE TITLE
LMenu selection triggers mouse events in LAxis behind it

### ISSUE THREAD

**Issue by yha** [JuliaPlots/Makie.jl#777]
Example:
```
scene, layout = layoutscene()
layout[1,1] = LMenu(scene; options=string.(1:20))
layout[2,1] = ax = LAxis(scene)
lines!(ax, rand(10))
on(ax.scene.events.mousebuttons) do buttons
    @show buttons
end
pt = select_point(ax.scene)
scatter!(ax, pt)
scene
```
Selecting a menu item triggers the mouse down and mouse up events on axis, and moves the scatter point.
I think the menu selection and hiding the menu should only happen on mouse up, and both events (mouse up and down) shouldn't trigger on the `LAxis`.

**Comment by jkrumbiegel** [comment]
yeah that one's annoying, the problem is that the laxis registers clicks by area, so it doesn't know there are things in front of it. I haven't figured out yet if that can be distinguished
